Change: Make continuation error messages consistent. (#15328)
OpenTTD/OpenTTD: src/lang/english.txt

# Config errors

> STR_CONFIG_ERROR                                                :{WHITE}Error with the configuration file...

Reviewer: ```suggestion
STR_CONFIG_ERROR                                                :Error with the configuration file...
```

STR_CONFIG_ERROR_INVALID_GRF                                    :ignoring invalid NewGRF '{RAW_STRING}': {STRING}
STR_CONFIG_ERROR_INVALID_GRF_NOT_FOUND                          :not found
STR_CONFIG_ERROR_INVALID_GRF_UNSAFE                             :unsafe for static use
STR_CONFIG_ERROR_INVALID_GRF_SYSTEM                             :system NewGRF
STR_CONFIG_ERROR_INVALID_GRF_INCOMPATIBLE                       :incompatible to this version of OpenTTD
STR_CONFIG_ERROR_INVALID_GRF_UNKNOWN                            :unknown
STR_CONFIG_ERROR_INVALID_SAVEGAME_COMPRESSION_LEVEL             :compression level '{RAW_STRING}' is not valid
STR_CONFIG_ERROR_INVALID_SAVEGAME_COMPRESSION_ALGORITHM         :savegame format '{RAW_STRING}' is not available. Reverting to '{RAW_STRING}'
STR_CONFIG_ERROR_INVALID_BASE_GRAPHICS_NOT_FOUND                :ignoring Base Graphics set '{RAW_STRING}': not found
STR_CONFIG_ERROR_INVALID_BASE_SOUNDS_NOT_FOUND                  :ignoring Base Sounds set '{RAW_STRING}': not found
STR_CONFIG_ERROR_INVALID_BASE_MUSIC_NOT_FOUND                   :ignoring Base Music set '{RAW_STRING}': not found

# Video initalization errors

> STR_VIDEO_DRIVER_ERROR                                          :{WHITE}Error with video settings...

Reviewer: ```suggestion
STR_VIDEO_DRIVER_ERROR                                          :Error with video settings...
```

STR_CONTENT_DETAIL_DEPENDENCIES                                 :{SILVER}Dependencies: {WHITE}{RAW_STRING}
STR_CONTENT_DETAIL_TAGS                                         :{SILVER}Tags: {WHITE}{RAW_STRING}

> STR_CONTENT_NO_ZLIB                                             :{WHITE}OpenTTD is built without "zlib" support...

Reviewer: ```suggestion
STR_CONTENT_NO_ZLIB                                             :OpenTTD is built without "zlib" support...
```

# Content downloading error messages
STR_CONTENT_ERROR_COULD_NOT_CONNECT                             :{WHITE}Could not connect to the content server...

> STR_CONTENT_ERROR_COULD_NOT_DOWNLOAD                            :{WHITE}Downloading failed...

Reviewer: ```suggestion
STR_CONTENT_ERROR_COULD_NOT_CONNECT                             :Could not connect to the content server...
STR_CONTENT_ERROR_COULD_NOT_DOWNLOAD                            :Downloading failed...
```

> STR_WARNING_LOADGAME_REMOVED_TRAMS                              :{WHITE}Game was saved in version without tram support. All trams have been removed

Reviewer: ```suggestion
STR_WARNING_LOADGAME_REMOVED_TRAMS                              :Game was saved in version without tram support. All trams have been removed
```

STR_MESSAGE_HEIGHTMAP_SUCCESSFULLY                              :{WHITE}Heightmap successfully saved as '{RAW_STRING}'. Highest peak is {NUM}

> STR_MESSAGE_SCREENSHOT_SUCCESSFULLY                             :{WHITE}Screenshot successfully saved as '{RAW_STRING}'

Reviewer: ```suggestion
STR_MESSAGE_HEIGHTMAP_SUCCESSFULLY                              :Heightmap successfully saved as '{RAW_STRING}'. Highest peak is {NUM}
STR_MESSAGE_SCREENSHOT_SUCCESSFULLY                             :Screenshot successfully saved as '{RAW_STRING}'
```

Author: There's loads more like this which are error messages but not as part of the Command system. I'd prefer to make a separate PR for these.
